test/Npgsql.Tests/ConnectionTests.cs

```diff
@@ -54,7 +54,7 @@
             Assert.That(conn.FullState, Is.EqualTo(ConnectionState.Open));
             Assert.That(conn.State, Is.EqualTo(ConnectionState.Open));
 
-            conn.Close();
+            await conn.CloseAsync();
```

Comment:
This method is otherwise fully synchronous, please revert

Reply:
line 47 is `using (var reader = await cmd.ExecuteReaderAsync())`. happy to make it fully async or sync if one of those is your preference

Reply:
Good point... OK to leave as-is, at some point we should do a more thorough pass and make sure everything is consistent.